sklearn/feature_selection/rfe.py

```diff
@@ -44,6 +44,9 @@ class RFE(BaseEstimator, MetaEstimatorMixin, SelectorMixin):
         algorithms such as Support Vector Classifiers and Generalized
         Linear Models from the `svm` and `linear_model` modules.
 
+        The estimator can also have a feature_importances_ attribute as an array. 
```

Comment:
I would already mention this in the first or second sentence. Maybe "A supervised learning estimator that determines feature importances. Feature importances are measured using either a ``coef_`` or ``feature_importances_`` attribute. [and then explain how each of them works]. You should put double backticks about code literals like ``feature_importances_``. (this docstring doesn't use this correctly)